ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/termsandconditions/v1/termsandconditions/termsandconditions.html

```diff
@@ -35,27 +35,29 @@
         <div data-sly-call="${label.label @componentId=tnc.id, labelValue=tnc.label.value, labelVisible=tnc.label.visible, labelRichText=tnc.label.richText, bemBlock='cmp-adaptiveform-termsandcondition'}" data-sly-unwrap></div>
         <div data-sly-call="${questionMark.questionMark @componentId=tnc.id, longDescription=tnc.description, bemBlock='cmp-adaptiveform-termsandcondition'}" data-sly-unwrap></div>
     </div>
-    <div class="cmp-adaptiveform-termsandcondition__content-container ${!wcmmode.edit && tnc.showAsPopup ? 'cmp-adaptiveform-termsandcondition__content-container--modal' : ''}"
-         data-cmp-visible="${tnc.showAsPopup ? 'false' : 'true'}">
-        <div class="cmp-adaptiveform-termsandcondition__body">
-            <div class="cmp-adaptiveform-termsandcondition__header" data-sly-test="${tnc.showAsPopup && !wcmmode.edit}">
-                <button type="button" class="cmp-adaptiveform-termsandcondition__close-button" >
-                    X
-                </button>
-                <h3>
-                    ${"Please review the terms and conditions" @ i18n, context='html'}
-                </h3>
-            </div>
-            <div class="cmp-adaptiveform-termsandcondition__content">
-                <div data-sly-test="${!tnc.showLink}" class="cmp-adaptiveform-termsandcondition__text">
-                    <div  data-sly-resource="${'text'}"></div>
-                    <div class="cmp-adaptiveform-termsandcondition__text-intersect"></div>
+    <div class="cmp-adaptiveform-termsandcondition__widget-container">
```

Comment:
Why is it necessary to add a new container element? Is it not possible with the existing container element `cmp-adaptiveform-termsandcondition__content-containe`?